app/actions/balance-actions.js

```diff
@@ -272,3 +275,38 @@ export const goToSend = (currencyId, balances) => (dispatch, getState) => {
 
 	return true;
 };
+
+export const totalFreezeSum = (frozenBalances) => {
+	let totalAmount = '0';
+	for (const fBalance in frozenBalances) {
+		if (frozenBalances[fBalance].balance) {
+			totalAmount = new BN(totalAmount).plus(new BN(frozenBalances[fBalance].balance.amount).div(FREEZE_BALANCE_PERCISION));
```

Comment:
`totalAmount.plus(` ...

Reply:
use it `.div(FREEZE_BALANCE_PERCISION)`  at the end of the function, not in middle